libres_util/src/res_log.c

```diff
@@ -113,7 +113,10 @@ void res_log_add_message(message_level_type message_level , FILE * dup_stream ,
  * required arguments to the string-formatting.
  */
 void res_log_add_fmt_message(message_level_type message_level , FILE * dup_stream , const char * fmt , ...) {
-    if (log_include_message(logh,message_level)) {
+  if(logh==NULL)
```

Comment:
Am I right that this could indeed result in logging being done in two different files (default first, and then after the log is initialized the file specified in the config)? If so, maybe we could add a log line in the 'res_log_init_log` function that warns that this behaviour can happen if `logh` is non-NULL upon initialization and just print the previous log-file in the new log_file.

I can imagine that this will be very easy to forget when debugging and will be extremely frustrating!

Reply:
Yes - this is correct; that is the way it has always been - and I agree we should inform about it. Not part of this PR though?!